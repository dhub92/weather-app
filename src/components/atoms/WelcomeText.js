import React from 'react';
import { grey } from '@material-ui/core/colors';
import { Typography, makeStyles } from '@material-ui/core';

const WelcomeText = () =>{
    const titleStyle = makeStyles({
        root:{
            fontSize:50,
            fontWeight: 200,
            color:grey[800]
        }
    })();

    const subTitleStyle = makeStyles({
        root:{
            fontSize:30,
            fontWeight: 200,
            color:grey[700]
        }
    })();

    return(<>
        <Typography className={titleStyle.root}>Welcome to the weather app</Typography>
        <Typography className={subTitleStyle.root}>Select a city to show its weather</Typography>
    </>)
}

export default WelcomeText;