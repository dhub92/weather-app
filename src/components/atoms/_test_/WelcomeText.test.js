import React from 'react';
import renderer from 'react-test-renderer';
import WelcomeText from '../WelcomeText';

it('renders correctly', () => {
    const tree = renderer.create(<WelcomeText />).toJSON();
    expect(tree).toMatchSnapshot();
  });