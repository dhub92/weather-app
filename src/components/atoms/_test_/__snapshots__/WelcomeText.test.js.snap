// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <p
    className="MuiTypography-root makeStyles-root-1 MuiTypography-body1"
  >
    Welcome to the weather app
  </p>,
  <p
    className="MuiTypography-root makeStyles-root-2 MuiTypography-body1"
  >
    Select a city to show its weather
  </p>,
]
`;
